PyInstaller/hooks/hook-PyQt5.QtWebEngineWidgets.py

```diff
@@ -21,17 +22,16 @@
 if compat.is_darwin:
     # This is based on the layout of the Mac wheel from PyPi.
     data_path = pyqt5_library_info.location['DataPath']
-    resources = ['lib', 'QtWebEngineCore.framework', 'Resources']
-    web_engine_process = ['lib', 'QtWebEngineCore.framework', 'Helpers']
-    # When Python 3.4 goes EOL (see
-    # `PEP 448 <https://www.python.org/dev/peps/pep-0448/>`_, this is
-    # better written as ``os.path.join(*rel_data_path, *resources[:-1])``.
-    datas += collect_system_data_files(
-        os.path.join(data_path, *resources),
-        os.path.join(*(rel_data_path + resources[:-1])), True)
-    datas += collect_system_data_files(
-        os.path.join(data_path, *web_engine_process),
-        os.path.join(*(rel_data_path + web_engine_process[:-1])), True)
+    libraries = ['QtCore', 'QtWebEngineCore', 'QtQuick', 'QtQml', 'QtNetwork',
+                 'QtGui', 'QtWebChannel', 'QtPositioning']
+    for i in libraries:
+        datas += collect_system_data_files(
+            os.path.join(data_path, 'lib', i + '.framework'),
+            os.path.join(*(rel_data_path + ['lib'])), True)
```

Comment:
Is this necessary for everything but QtWebEngineCore and QtWebChannel? If so, this should probably go in other hooks.

Reply:
For some reason, QtWebEngineProcess needs this: https://github.com/pyinstaller/pyinstaller/issues/2276#issuecomment-280919265
I verified that these files are required for QTWE to run.

Reply:
OK, thanks for checking.